react/features/base/avatar/components/Avatar.js

```diff
@@ -208,7 +220,7 @@ class Avatar<P: Props> extends PureComponent<P, State> {
             avatarProps.url = effectiveURL;
         }
 
-        const initials = getInitials(_initialsBase);
+        const initials = ignoreInitials ? null : getInitials(_initialsBase);
```

Comment:
Why do we want to ignore the initials? It would be nice to get the avatar color to match the participant it's related to I guess.

Reply:
I got from UX feedback that we want only one color to be displayed for placeholder with the icon in the middle, no initials on it; I will raise this and if we want to match the colors of the participant, I would need to only skip this part `avatarProps.initials = initials;`